doc/runtime-support-policy.md

```diff
@@ -0,0 +1,42 @@
+# Node.js
+
+## Version Support Policy
+
+We try to support two (2) Node.js versions up to the current _Active LTS_ version.
+
+> For example:
+> - Node.js 14 is _Active LTS_: we'll support both 14 and 12 and drop 10.
+> - Node.js 16 is _Active LTS_: we'll support both 16 and 14 and drop 12.
+
+See https://nodejs.org/en/about/releases/ to see the status of Node.js versions.
+
+We recommend setting up your environments to run Theia using the Node.js _Active LTS_, but any supported version should work as well. Fill an issue otherwise: https://github.com/eclipse-theia/theia/issues/new/choose.
```

Comment:
```suggestion
We recommend setting up your environment to run Theia using the Node.js _Active LTS_, but any supported version should work as well. Fill an issue otherwise: https://github.com/eclipse-theia/theia/issues/new/choose.
```